mono/sgen/sgen-memory-governor.c

```diff
@@ -235,6 +235,11 @@ sgen_memgov_major_post_sweep (mword used_slots_size)
 
 		sgen_add_log_entry (log_entry);
 	}
+
+	sgen_gc_info.heap_size_bytes = sgen_major_collector.get_num_major_sections () * sgen_major_collector.section_size + sgen_los_memory_usage_total;
+	sgen_gc_info.fragmented_bytes = sgen_gc_info.heap_size_bytes - sgen_los_memory_usage - (used_slots_size + sgen_total_allocated_major - total_allocated_major_end);
+	sgen_gc_info.memory_load_bytes = sgen_los_memory_usage + used_slots_size + sgen_total_allocated_major - total_allocated_major_end;
```

Comment:
This refers to the total system not just sgen memory.

Reply:
This API is supposed to give the total system memory, right? Not just sgen.

I named the variable sgen_gc_info because its used in the sgen implementation of this API, not because it is only tracking memory used by sgen. But I could rename it if that is not clear.

Reply:
Memory load is for the whole system. I don't have any naming complaints or whether these stats are only produced inside sgen, even though they are not 100% sgen related.